client/README.md

```diff
@@ -65,7 +63,9 @@ export BQMS_GCS_BUCKET="<YOUR_GCS_BUCKET>"
 
 ## Installation
 
-Prerequisites: Python >= 3.7.2.
+Prerequisites:
+- Python >= 3.7.2.
+- package `pkg-config` and `libicu-dev`
```

Comment:
@davidkhala Can you create a separate PR for this as this should be merged asap? Thanks.

Reply:
@ajwelch4 good point, will do.